kotsadm/pkg/handlers/config.go

```diff
@@ -141,6 +155,103 @@ func UpdateAppConfig(w http.ResponseWriter, r *http.Request) {
 	JSON(w, 200, UpdateAppConfigResponse{Success: true})
 }
 
+func LiveAppConfig(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+	w.Header().Set("Access-Control-Allow-Headers", "content-type, origin, accept, authorization")
+
+	if r.Method == "OPTIONS" {
+		w.WriteHeader(200)
+		return
+	}
+
+	liveAppConfigResponse := LiveAppConfigResponse{
+		Success: false,
+	}
+
+	liveAppConfigRequest := LiveAppConfigRequest{}
+	if err := json.NewDecoder(r.Body).Decode(&liveAppConfigRequest); err != nil {
+		logger.Error(err)
+		liveAppConfigResponse.Error = "failed to decode request body"
```

Comment:
If these messages are not including the original error intentionally, there should be a comment about it.  Because someone will change it.

Reply:
This is the pattern through the entire file, though
these errors are supposed to be pretty